tests/statuspage_test.go

```diff
@@ -0,0 +1,43 @@
+package tests
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"testing"
+	model "web/model"
+	httpTemplates "web/views/templates"
+
+	"github.com/jarcoal/httpmock"
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStatusPageWhenAllServicesDown(tests *testing.T) {
```

Comment:
Also, I would add comments to the tests where `Arrange, Act, Assert` starts.
https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/